Examples/PublisherExampleSwiftUI/PublisherExampleSwiftUI/ViewModel/MapViewModel.swift

```diff
@@ -49,7 +49,11 @@ class MapViewModel: ObservableObject {
         
         let connectionConfiguration = ConnectionConfiguration(apiKey: EnvironmentHelper.ABLY_API_KEY, clientId: "Asset Tracking Publisher Example")
         let resolution = Resolution(accuracy: .balanced, desiredInterval: 5000, minimumDisplacement: 100)
-        
+                
+        /**
+         For simulated location add `locationSource(.init(locationSource: SimulatedLocations.recordedLocations()))` for
+         PublisherFactory configuration chain
+         */
         publisher = try! PublisherFactory.publishers()
```

Comment:
I'm wondering about two things:
- can we add a switch somewhere in the UI that would allow the example app user to choose if he wants to use the real location or the recorded ones (this feels a bit out of the scope of this PR though :sweat_smile: )
- maybe let's change this to a comment somewhere in the configuration chain that would enable the recorded location if you simply uncomment one line? :thinking: Something like:
```swift
    .log(LogConfiguration())
//    Uncomment below line to enable simulated location
//    .locationSource(.init(locationSource: SimulatedLocations.recordedLocations()))
    .routingProfile(.driving)
```

Reply:
> can we add a switch somewhere in the UI that would allow the example app user to choose if he wants to use the real location or the recorded ones (this feels a bit out of the scope of this PR though 😅 )

exactly, this is out of the scope of this PR

> maybe let's change this to comment somewhere in the configuration chain that would enable the recorded location if you simply uncomment one line? 🤔 Something like

f90503d203efbd281f65eded41e3832aeb189345